examples/eth-config/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 RealSense, Inc. All Rights Reserved.
+
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Please update to :
cmake_minimum_required(VERSION 3.10)
Like other cmake lists.
see: #14219